### PR TITLE
Auto-annotation support for jobs

### PR DESCRIPTION
**Pull request by zhiltsov-max**
### Motivation and context

Depends on #6204 

- Added an option to run auto-annotation on jobs
- Added API schema definitions for offline lambda call endpoints

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Review by azhavoro — APPROVED** [review]

File: cvat/schema.yml [hunk 1]
```diff
@@ -6540,6 +6560,98 @@ components:
...
+          type: string
+          format: date-time
+        started:
+          type: string
+          format: date-time
+        ended:
+          type: string
+          format: date-time
+        exc_info:
+          type: string
```
Comment (SpecLad) [inline cvat/schema.yml:6586]: I think these should be marked nullable (I see that they are set to null in the response to the initial create request).

File: cvat/schema.yml [hunk 2]
```diff
@@ -6540,6 +6560,98 @@ components:
...
+        id:
+          type: string
+          description: The name of the function
+        task:
+          type: integer
+          description: The id of the task
+        job:
+          type: integer
+          description: The id of the job
+        threshold:
```
Comment (SpecLad) [inline cvat/schema.yml:6607]: This should be marked nullable, as well.

File: cvat/schema.yml (hunk 1 above)
Comment (zhiltsov-max) [reply cvat/schema.yml]: I couldn't find where it happens for `exc_info`, but there are other fields that can potentially be null, so I marked them.

File: cvat/schema.yml (hunk 2 above)
Comment (zhiltsov-max) [reply cvat/schema.yml:6614]: Done

**Review by SpecLad — APPROVED** [review]

**Pull request merged**